__tests__/lint/spec-strict-refs/snapshot.js

```diff
@@ -1,46 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`E2E lint spec-ref-validation 1`] = `
-
-validating /openapi.yaml...
-[1] openapi.yaml:6:3 at #/info/$ref
-
-Field $ref is not expected here.
-
-4 |     description: Default server
-5 | info:
-6 |   $ref: './info.yaml'
-  |   ^^^^
-7 | paths:
-8 |   /pet:
-
-Error was generated by the spec-strict-refs rule.
-
-
-[2] openapi.yaml:20:19 at #/paths/~1pet/get/responses/200/content/application~1json/schema/properties/$ref
-
-Field $ref is not expected here.
-
-18 | type: object
-19 | properties:
-20 |   $ref: './props.yaml'
-   |   ^^^^
-21 |   name:
-22 |     type: string
-
-Error was generated by the spec-strict-refs rule.
-
+exports[`E2E lint spec-strict-refs 1`] = `
```

Comment:
Why this has been changed?

Reply:
@tatomyr `E2E lint spec-ref-validation 1` from old snapshot. and this one  just changed location from [here](https://github.com/Redocly/redocly-cli/pull/1133/files?diff=split&w=0#diff-3fdf21f2551fc727676e37674367dfa62ed3434527db5aecfe04a9daf270a696L42)